config/locales/forms/fr.yml

```diff
@@ -108,6 +108,7 @@ fr:
           ou contactez votre administrateur pour vous assurer que votre certificat
           est à jour.
       choose_different_certificate: Choisissez un certificat différent
+      nickname: Pseudo carte PIV/CAC
```

Comment:
Question! I completely made up es/fr translations (since it looks like we aren't using placeholders these days) -- can somebody let me know what the translation process is for one-offs like this? (I only found docs for bulk translations)

Reply:
We have a glossary for common translated terms in the content guide here: https://design.login.gov/content/

And for small things like this we lean on Google Translate. For larger blocks of text we'll go through the full translation process.

Reply:
Sweet, thanks!